### PR TITLE
Deploy csv updates for recent glider turns

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
515_recovery
387_recovery
379_recovery

**Comment by cdobs** [comment]
[3202-01102_Irminger_11_QLR_2024-06-24_SNW_RGT.pdf](https://github.com/user-attachments/files/15992207/3202-01102_Irminger_11_QLR_2024-06-24_SNW_RGT.pdf)
[Papa_QLR-Mooring Tables.docx](https://github.com/user-attachments/files/15992208/Papa_QLR-Mooring.Tables.docx)

File: deployment/CP15MOAS-GL379_Deploy.csv [hunk 1]
```diff
@@ -5,9 +5,9 @@ AR82,,AR82,,CP15MOAS-GL379-02-FLORTM000,1,1,2024-04-04T22:44:00,2024-04-18T18:30
...
\ No newline at end of file
+KH20240523,,KH20240609,,CP15MOAS-GL379-00-ENG000000,2,1,2024-05-23T13:32:00,2024-06-09T14:27:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,35.92208,-74.75223,,1000,1000,OM,,
+KH20240523,,KH20240609,,CP15MOAS-GL379-01-ADCPAM000,2,1,2024-05-23T13:32:00,2024-06-09T14:27:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49016,35.92208,-74.75223,,1000,1000,OM,,
+KH20240523,,KH20240609,,CP15MOAS-GL379-02-FLORTM000,2,1,2024-05-23T13:32:00,2024-06-09T14:27:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-04194,35.92208,-74.75223,,1000,1000,OM,,
+KH20240523,,KH20240609,,CP15MOAS-GL379-03-CTDGVM000,2,1,2024-05-23T13:32:00,2024-06-09T14:27:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09030,35.92208,-74.75223,,1000,1000,OM,,
+KH20240523,,KH20240609,,CP15MOAS-GL379-04-DOSTAM000,2,1,2024-05-23T13:32:00,2024-06-09T14:27:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00496,35.92208,-74.75223,,1000,1000,OM; top mount,,
+KH20240523,,KH20240609,,CP15MOAS-GL379-05-PARADM000,2,1,2024-05-23T13:32:00,2024-06-09T14:27:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50220,35.92208,-74.75223,,1000,1000,OM,,
```
Comment (rtravis-whoi) [inline deployment/CP15MOAS-GL379_Deploy.csv:13]: looks good

File: deployment/CP15MOAS-GL387_Deploy.csv [hunk 2]
```diff
@@ -5,9 +5,9 @@ AR82,,KH20240516,,CP15MOAS-GL387-02-FLORTM000,1,1,2024-04-05T17:51:00,2024-05-16
...
\ No newline at end of file
+KH20240523,,KH20240616,,CP15MOAS-GL387-00-ENG000000,2,1,2024-05-23T15:08:00,2024-06-16T18:42:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGCON-GLDRCP-00387,35.94993,-74.79575,,350,350,MA,,
+KH20240523,,KH20240616,,CP15MOAS-GL387-01-ADCPAM000,2,1,2024-05-23T15:08:00,2024-06-16T18:42:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-55979,35.94993,-74.79575,,350,350,MA,,
+KH20240523,,KH20240616,,CP15MOAS-GL387-02-FLORTM000,2,1,2024-05-23T15:08:00,2024-06-16T18:42:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-04471,35.94993,-74.79575,,350,350,MA,,
+KH20240523,,KH20240616,,CP15MOAS-GL387-03-CTDGVM000,2,1,2024-05-23T15:08:00,2024-06-16T18:42:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09059,35.94993,-74.79575,,350,350,MA,,
+KH20240523,,KH20240616,,CP15MOAS-GL387-04-DOSTAM000,2,1,2024-05-23T15:08:00,2024-06-16T18:42:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00310,35.94993,-74.79575,,350,350,MA; top mount,,
+KH20240523,,KH20240616,,CP15MOAS-GL387-05-PARADM000,2,1,2024-05-23T15:08:00,2024-06-16T18:42:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50151,35.94993,-74.79575,,350,350,MA,,
```
Comment (rtravis-whoi) [inline deployment/CP15MOAS-GL387_Deploy.csv:13]: looks good

File: deployment/GI05MOAS-PG566_Deploy.csv [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+AR84-01,,,,GI05MOAS-PG566-00-ENG000000,4,1,2024-06-16T20:13:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00566,CGCON-GLDRGP-00566,59.90728,-39.27502,,200,200,,,
+AR84-01,,,,GI05MOAS-PG566-01-CTDGVM000,4,1,2024-06-16T20:13:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-CTDGVM-09295,59.90728,-39.27502,,200,200,,,
+AR84-01,,,,GI05MOAS-PG566-02-DOSTAM000,4,1,2024-06-16T20:13:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-DOSTAM-00361,59.90728,-39.27502,,200,200,top mount,,
+AR84-01,,,,GI05MOAS-PG566-03-FLORTM000,4,1,2024-06-16T20:13:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-FLORTM-04089,59.90728,-39.27502,,200,200,,,
+AR84-01,,,,GI05MOAS-PG566-04-FLORTO000,4,1,2024-06-16T20:13:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-FLORTO-01347,59.90728,-39.27502,,200,200,,,
+AR84-01,,,,GI05MOAS-PG566-05-NUTNRM000,4,1,2024-06-16T20:13:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-NUTNRM-00662,59.90728,-39.27502,,200,200,,,
+AR84-01,,,,GI05MOAS-PG566-06-PARADM000,4,1,2024-06-16T20:13:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00566,CGINS-PARADM-50202,59.90728,-39.27502,,200,200,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-PG566_Deploy.csv:15]: looks good

File: deployment/GP05MOAS-PG515_Deploy.csv [hunk 4]
```diff
@@ -6,10 +6,10 @@ TN323,,TY-1402,,GP05MOAS-PG515-03-FLORTM000,2,1,2015-06-02T09:55:00,2016-02-17T2
...
+SKQ202308S,,AT50-25,,GP05MOAS-PG515-01-CTDGVM000,5,1,2023-05-16T15:12:00,2024-06-13T00:00:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-CTDGVM-09231,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,AT50-25,,GP05MOAS-PG515-02-DOSTAM000,5,1,2023-05-16T15:12:00,2024-06-13T00:00:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-DOSTAM-00192,50.46368,-144.85315,,200,200,top mount,,
+SKQ202308S,,AT50-25,,GP05MOAS-PG515-03-FLORTM000,5,1,2023-05-16T15:12:00,2024-06-13T00:00:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTM-03501,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,AT50-25,,GP05MOAS-PG515-04-FLORTO000,5,1,2023-05-16T15:12:00,2024-06-13T00:00:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTO-01239,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,AT50-25,,GP05MOAS-PG515-05-NUTNRM000,5,1,2023-05-16T15:12:00,2024-06-13T00:00:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-NUTNRM-00760,50.46368,-144.85315,,200,200,,,
+SKQ202308S,,AT50-25,,GP05MOAS-PG515-06-PARADM000,5,1,2023-05-16T15:12:00,2024-06-13T00:00:00,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-PARADM-50213,50.46368,-144.85315,,200,200,,,
```
Comment (rtravis-whoi) [inline deployment/GP05MOAS-PG515_Deploy.csv:15]: looks good

File: deployment/GP05MOAS-PG563_Deploy.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+SKQ202408S,,,,GP05MOAS-PG563-00-ENG000000,2,1,2024-06-01T15:07:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00563,CGCON-GLDRGP-00563,50.52637,-144.81762,,200,200,,,
+SKQ202408S,,,,GP05MOAS-PG563-01-CTDGVM000,2,1,2024-06-01T15:07:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-CTDGVM-09286,50.52637,-144.81762,,200,200,,,
+SKQ202408S,,,,GP05MOAS-PG563-02-DOSTAM000,2,1,2024-06-01T15:07:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-DOSTAM-00489,50.52637,-144.81762,,200,200,top mount,,
+SKQ202408S,,,,GP05MOAS-PG563-03-FLORTM000,2,1,2024-06-01T15:07:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-FLORTM-03814,50.52637,-144.81762,,200,200,,,
+SKQ202408S,,,,GP05MOAS-PG563-04-FLORTO000,2,1,2024-06-01T15:07:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-FLORTO-01331,50.52637,-144.81762,,200,200,,,
+SKQ202408S,,,,GP05MOAS-PG563-05-NUTNRM000,2,1,2024-06-01T15:07:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-NUTNRM-00660,50.52637,-144.81762,,200,200,,,
+SKQ202408S,,,,GP05MOAS-PG563-06-PARADM000,2,1,2024-06-01T15:07:00,,CGVGP-05MOAS-00000,CGVEH-GLDRGP-00563,CGINS-PARADM-50199,50.52637,-144.81762,,200,200,,,
```
Comment (rtravis-whoi) [inline deployment/GP05MOAS-PG563_Deploy.csv:8]: looks good

File: deployment/GI05MOAS-PG566_Deploy.csv (hunk 3 above)
Comment (snwhite) [reply deployment/GI05MOAS-PG566_Deploy.csv:15]: Looks good.

File: deployment/CP15MOAS-GL379_Deploy.csv (hunk 1 above)
Comment (snwhite) [reply deployment/CP15MOAS-GL379_Deploy.csv:13]: Looks good.

File: deployment/CP15MOAS-GL387_Deploy.csv (hunk 2 above)
Comment (snwhite) [reply deployment/CP15MOAS-GL387_Deploy.csv:13]: Looks good.

File: deployment/GP05MOAS-PG515_Deploy.csv (hunk 4 above)
Comment (snwhite) [reply deployment/GP05MOAS-PG515_Deploy.csv:15]: Looks good.

File: deployment/GP05MOAS-PG563_Deploy.csv (hunk 5 above)
Comment (snwhite) [reply deployment/GP05MOAS-PG563_Deploy.csv:8]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**